doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -42,7 +42,7 @@ These updates appear in detail in the sections below. Many also appear in
 
 .. _The crop model:
 
-The crop model
+The crop model: stable and bioenergy crops
```

Comment:
Do you mean “staple” instead of “stable”? One thing to consider with this distinction is that crops like cotton, which are used in CLM5 are not necessarily staple crops. I don’t currently have a great suggestion for how to distinguish these — perhaps “cash crops”? Although would biofuels also be considered a cash crop? I'm not sure if you can think of a good distinction, but I'm happy to brainstorm if you'd like.